tools/wikipathways/.shed.yml

```diff
@@ -0,0 +1,20 @@
+categories:
+  - Pathway Analysis
+description: |
+  WikiPathways API tools
+long_description: |
+  WikiPathways is a database of biological pathways maintained by and for the scientific community. 
+name: wikipathways
+owner: erasmus-medical-center
+remote_repository_url: https://github.com/ErasmusMC-Bioinformatics/galaxytools-emc/tree/master/tools/wikipathways
+homepage_url: https://www.wikipathways.org/index.php/WikiPathways
+type: unrestricted
+auto_tool_repositories:
+  name_template: "{{ tool_id }}"
+  description_template: "Wrapper for WikiPathways: {{ tool_name }}"
+suite:
```

Comment:
did you do this because you are expecting more tools here or?

Reply:
For now it is only one tool, but I thought as we're working on it now I would expect that more things are going to be wrapped sooner or later. Anyway I can change it back to a single tool.

Reply:
ah yeah. Even if it is multiple tools, it doesn't have to be a suite (they have some drawbacks), but we can decide on this whenever we actually get more tools